### PR TITLE
Moving utils from bootstrap to here.

### PR DESCRIPTION
**Pull request by cyberswat**

### CODE REVIEW

File: utils/system.go [hunk 1]
```diff
@@ -2,13 +2,24 @@ package utils
...
 	"os"
 	"os/exec"
 	"strings"
 
 	log "github.com/Sirupsen/logrus"
 )
 
+var (
+	// DrudBin is the name of the DRUD binary
+	DrudBin = "drud"
```
Comment (beeradb) [inline utils/system.go]: None of these feels like they should be here to me

File: utils/string.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+func RandStringRunes(n int) string {
+	b := make([]rune, n)
+	for i := range b {
+		b[i] = letterRunes[rand.Intn(len(letterRunes))]
+	}
+	return string(b)
+}
+
+// RandomString returns a random string of len strlen
+func RandomString(strlen int) string {
```
Comment (beeradb) [inline utils/string.go]: I already did this in my PR (https://github.com/drud/drud-go/pull/26/files) which is 4 days old. It'd be nice to get that merged first imho

Comment (cyberswat) [inline utils/string.go]: merged yours and updated this pull request

Comment (beeradb) [inline utils/string.go]: mine was in strings.go.. are we sure this compiles after being updated?

Comment (cyberswat) [inline utils/string.go]: Should.  Updating vendor stuff in bootstrap to run test .. let you know in just a bit

File: utils/system.go (hunk 1 above)
Comment (beeradb) [inline utils/system.go]: Const in the packages where tests live? The drudbin I'm less concerned about than the test apps/test deploys. Doesn't feel like this sort of code should be in a generic package.

**Comment by beeradb** [comment]
This doesn't work for me after pulling it down locally. There appear to be a couple issues with it:

1. strings.go and string.go both contain a RandomString function. This can just be resolved by doing a git rm on strings.go
2. After resoving that, docker.go throws a bunch of errors re: an undefined docker package. I believe it's missing an import to github.com/fsouza/go-dockerclient.

I'm going to go ahead and push up these changes, but they probably need verification since I'm not the author of this code.

File: utils/system.go (hunk 1 above)
Comment (cyberswat) [inline utils/system.go]: Moved to testconfig_test.go

**Pull request merged**